src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/Maintainability/DoNotIgnoreMethodResults.cs

```diff
@@ -176,6 +187,10 @@ public override void Initialize(AnalysisContext analysisContext)
                                 {
                                     rule = PureMethodRule;
                                 }
+                                else if (targetMethod.ContainingType.Equals(enumerableType))
```

Comment:
There does not seem to be anything specific to linq methods in the implementation. Should we just update the rule name, message, etc. to use the term `Enumerable` instead of `Linq`?

Reply:
Ah, I missed `enumerableType` here is actually `SystemLinqEnumerable`. Can the local name be changed to `linqEnumerableType`?